tests/e2e/support/api/davSpaces/spaces.ts

```diff
@@ -4,6 +4,36 @@ import join from 'join-path'
 import { getPersonalSpaceId } from '../graph'
 import { config } from '../../../config'
 
+export const doesFolderExists = async ({
+  user,
+  parentFolder,
+  subFolder,
+  oc10Path
+}: {
+  user: User
+  parentFolder: string
+  subFolder: string
+  oc10Path: string
+}): Promise<boolean> => {
+  const getResponse = await request({
+    method: 'GET',
+    path: config.ocis
+      ? join(
+          'remote.php',
+          'dav',
+          'spaces',
+          await getPersonalSpaceId({ user }),
+          parentFolder,
+          subFolder
+        )
+      : oc10Path,
+    user: user,
+    formatJson: false
+  })
+
+  return getResponse.status !== 404
```

Comment:
return `true` only if the status is `200`, otherwise `false`
```suggestion
  return getResponse.status === 200
```